tests/jobs.py

```diff
@@ -901,80 +901,6 @@ def do_schedule(function, function2):
         do_schedule()
         self.assertEquals(2, len(executor.queued_tasks))
 
-    def test_retry_still_in_executor(self):
```

Comment:
I just added this test to resolve the issue that retries were not being executed (one of the blockers!). Tasks should retry before the dag_run reports itself as failed. This tests generates a dag_run with tasks that need to be retried before reporting the dag_run as failed. So I don't think this test should be removed.